### PR TITLE
add extra test for --include-easyblocks for generic easyblocks

### PR DESCRIPTION
**Pull request by boegel**
test case for an unintended yet very useful side-effect of a fallback mechanism in `get_easyblock_class`

pointed out by @gppezzi

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: test/framework/options.py [hunk 1]
```diff
@@ -1760,9 +1760,85 @@ def test_include_easyblocks(self):
...
+        os.remove(os.path.join(self.test_prefix, 'generic', 'foobar.py'))
+        reload(easybuild.easyblocks.generic)
+
+        error_msg = "Failed to obtain class for FooBar easyblock"
+        self.assertErrorRegex(EasyBuildError, error_msg, get_easyblock_class, 'FooBar')
+
+        # clear log
+        write_file(self.logfile, '')
+
+        # importing without specifying 'generic' also works, and generic easyblock can be imported as well
```
Comment (wpoely86) [inline test/framework/options.py]: extend this comment a bit more about why this works

Comment (boegel) [inline test/framework/options.py]: done

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**